extensions/jwks/README.md

```diff
@@ -0,0 +1,41 @@
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/sovity/edc-extensions">
+    <img src="https://raw.githubusercontent.com/sovity/edc-ui/main/src/assets/images/sovity_logo.svg" alt="Logo" width="300">
+  </a>
+
+<h3 align="center">EDC-Connector Extension:<br />JWKS</h3>
+
+  <p align="center">
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=bug_report.md">Report Bug</a>
+    ·
+    <a href="https://github.com/sovity/edc-extensions/issues/new?template=feature_request.md">Request Feature</a>
+  </p>
+</div>
+
+## About this Extension
+
+The JWKS-Extension provides an endpoint in the default API of the EDC-Connector, that returns the
+[JWKS]{https://datatracker.ietf.org/doc/html/rfc7517#section-5} of the connector. It can be accessed
+using the `:{WEB_HTTP_PORT}/{WEB_HTTP_PATH}/jwks` (default: `:11001/api/jwks`) endpoint.
```

Comment:
currently in the CE the (Default) API Group "Web HTTP" Port is only used for in-container health checks

Do we want to use the Protocol API or make people expose the (Default) API Group "Web HTTP"?

Reply:
In a sense that the protocol endpoint is already publicly available, I moved it to the protocol context. Note that it is now located by default under `:11003/api/dsp/jwks`